src/datasource/components/QueryEditor.test.tsx

```diff
@@ -88,7 +88,7 @@ const getDefaultProps = (grafanaLive: boolean) => ({
 });
 
 describe('QueryEditor', () => {
-  function runGrafanaLiveEnabled() {
+  function run() {
```

Comment:
What's this change about?

Reply:
It was originally accidentally named `runGrafanaLiveEnabled`, this function should be named just `run()`